src/types/user/user-interfaces/user.interfaces.ts

```diff
@@ -52,15 +60,18 @@ export interface UserGeneralInfo
 }
 
 export interface UpdateUserParams
-  extends Pick<
-    UserResponse,
-    | 'firstName'
-    | 'lastName'
-    | 'address'
-    | 'professionalSummary'
-    | 'nativeLanguage'
+  extends Partial<
+    Pick<
+      UserResponse,
+      | 'firstName'
+      | 'lastName'
+      | 'address'
+      | 'professionalSummary'
+      | 'nativeLanguage'
+      | 'professionalBlock'
+    >
```

Comment:
There are many types. Maybe it's better to move it to separate variable?

Reply:
Yes, it's pretty uncomfortable to read, thank you for you thought, fixed!